src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,437 @@
+| TubingType           | RECOMMENDED | [string][] | Description of the type of tubing used, ideally including the material and (internal) diameter.                                                                                                                      |
+| TubingLength         | RECOMMENDED | [number][] | The length of the blood tubing, from the subject to the detector in the default unit centimeter.                                                                                                              |
+| DispersionConstant   | RECOMMENDED | [number][] | External dispersion time constant resulting from tubing in default unit seconds.                                                                                                                                                                             |
+| Haematocrit            | RECOMMENDED | [number][] | Measured haematocrit, meaning the volume of erythrocytes divided by the volume of whole blood.                                                                                                                       |
+| BloodDensity                | RECOMMENDED | [number][] | Measured blood density. Unit of blood density should be in g/mL.                                                                                                                                                     |
+
+### Time
+
+If manual blood samples, for example, using a COBRA counter are available,
+then time information as detailed below should be added to the `*recording-manual_blood.tsv`
+and the values below should be added to the `*recording-manual_blood.json`.
+If blood samples using an autosampler, for example, using an Allogg autosampler are available,
+then time information as detailed below should be added to the `*recording-autosampler_blood.tsv`
+and the values below should be added to the `*recording-autosampler_blood.json`.
+
+| **Column name** | **Description** | **Units**                                      |
+| --------------- | ---------------------------------------------------- | ---------- | 
+| time | Time in relation to `TimeZero` defined by the `*_pet.json`. (for example, 5) | Unit of time steps (for example, "s") |       |                           |
+
+### Radioactivity in plasma
+
+This section may be omitted if plasma measurements of radioactivity were not made.
+It contains information regarding sampled plasma data.
+If plasma measurements are available by manual sampling, they should be added to the `*recording-blood_discrete.tsv`
+and the values below should be added to the `*recording-blood_discrete.json`.
+If plasma measurements are available by an autosampler, they should be added to the `*recording-blood_continous.tsv`
```

Comment:
I don't think there exist autosamplers which can measure plasma. This might be confusing.

Reply:
You are right, it is not the autosampler that measures plasma data. But there are cases where data from automated samples (whole blood) are centrifuged to collect plasma samples. See Parsey et al. 2006, JNM, metabolite consideration. These should go into recording-autosampler_blood.<extension>. 

Maybe the sentence could be updated to: "If plasma measurements are derived from continuous blood samples from an autosampler, they ...."? 

Similar answer to the comment below. Let me know what you (or others) think.

Reply:
Well, I stand corrected!  I've not seen this before - thanks for correcting me!  I thought that the small amounts of blood sampled by automatic sampling systems were too small for separating out the plasma and quantifying radioactivity. Though notable that they did do the metabolite analysis using the manual samples only.

I think the original wording is probably better then in this case, rather than "derived from continuous blood samples". If I remember correctly, at KI for manual samples at early time points, they would be collected from the blood efflux from the automatic sampler (over several seconds). So the time point would be slightly more dispersed than if it were sampled directly, but I this would still constitute a manual sample as it would be a much larger volume than the auto samples, and would be measured separately in the same gamma counter as the remainder of the manual samples.  So "derived from continuous blood samples" could be confusing in this context.

Reply:
So @mathesong, we agree to keep the wording then?

Reply:
Agree!